sql/hypertable.sql

```diff
@@ -1,3 +1,3 @@
 -- Trigger that blocks INSERTs on the hypertable's root table
-CREATE OR REPLACE FUNCTION _timescaledb_internal.insert_blocker() RETURNS trigger
+CREATE OR REPLACE FUNCTION _timescaledb_internal.ts_insert_blocker_on_hypertable() RETURNS trigger
```

Comment:
Not  sure  we need to rename this function. It is already namespaced by the schema and the `on_hypertable` suffix seems superfluous.